Missing dependency javax.activation

Hello,

I get the following error when compiling on Kali:

```
[ERROR] /opt/marshalsec/src/main/java/marshalsec/gadgets/JDKUtil.java:[52,24] package javax.activation does not exist
```

Adding the following block to the `<dependencies>` section of the pom.xml file does the trick:

```
<dependency>
    <groupId>javax.activation</groupId>
    <artifactId>activation</artifactId>
    <version>1.1.1</version>
</dependency>
```

Cheers!

--- a/pom.xml
+++ b/pom.xml
@@ -16,6 +16,12 @@
 
 		<!-- Util -->
 		<dependency>
+			<groupId>com.sun.activation</groupId>
+			<artifactId>javax.activation</artifactId>
+			<version>1.2.0</version>
+		</dependency>
+
+		<dependency>
 			<groupId>org.javassist</groupId>
 			<artifactId>javassist</artifactId>
 			<version>3.19.0-GA</version>

